dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/ZonedIntervalIterable.java

```diff
@@ -150,18 +134,28 @@ public static ZonedIntervalIterable forConnectorArguments(
           "At least one day of query logs should be exported; you specified: " + daysToExport);
 
     LOG.info(
-        "Log entries within the last {} days will be exported in increments of 1 {}.",
+        "Log entries within the last {} days will be exported in increments of {} seconds.",
         daysToExport,
-        timeUnit);
+        DurationFormatUtils.formatDuration(duration.toMillis(), "**H:mm:ss**", true));
 
-    return ZonedIntervalIterable.forTimeUnitsUntilNow(unitsInADay * daysToExport, timeUnit);
+    int chunksInADay = Math.toIntExact(Duration.ofDays(1).getSeconds() / duration.getSeconds());
```

Comment:
Optional: I would suggest to change the type to `long` instead of casting to `int`.